adapters/eplanning/eplanning.go

```diff
@@ -61,16 +65,62 @@ type hbResponseAd struct {
 	Height       uint64 `json:"h,omitempty"`
 }
 
+func isPriority(index1 int, index2 int) bool {
+	if index1 > -1 {
+		if index2 > -1 {
+			if index1 < index2 {
+				return false
+			} else {
+				return true
+			}
+		} else {
+			return false
+		}
+	} else {
+		if index2 > -1 {
+			return true
+		} else {
+			return false
+		}
+	}
+}
+
+func lessByPriority(w1, h1, w2, h2 uint64, priorityOrderForSizesAsc []string) bool {
+	size1 := fmt.Sprintf("%dx%d", w1, h1)
```

Comment:
I'm a bit concerned there are 2 `Sprintf` calls occurring for each compare. Might not be a big deal if there aren't many formats though. How many formats are common on requests to your adapter?